### PR TITLE
[Feat]: [CCM-7668]: Anomaly detection for azure

### PR DESCRIPTION
**Pull request by hitesharinga**
You can use the following comments to re-trigger PR Checks

- Compile: `trigger compile`
- runAeriformCheck: `trigger AeriformCheck`
- CodeFormat: `trigger codeformat`
- MessageMetadata: `trigger messagecheck`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- runDockerizationCheck: `trigger dockerizationcheck`
- runAuthorCheck: `trigger authorcheck`
- Checkstyle: `trigger checkstyle`
- PMD: `trigger pmd`
- TI-bootstrap: `trigger ti0`
- TI-bootstrap1: `trigger ti1`
- TI-bootstrap2: `trigger ti2`
- TI-bootstrap3: `trigger ti3`
- TI-bootstrap4: `trigger ti4`
- FunctionalTest1: `trigger ft1`
- FunctionalTest2: `trigger ft2`
- CodeBaseHash: `trigger codebasehash`

---
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/33224)

### CODE REVIEW

File: 280-batch-processing/src/main/java/io/harness/batch/processing/anomalydetection/RemoveDuplicateAnomaliesTasklet.java [hunk 1]
```diff
@@ -44,9 +44,9 @@ public RepeatStatus execute(StepContribution stepContribution, ChunkContext chun
     List<String> duplicateIds = getDuplicates(anomalyList);
     log.info("[RDA] deleting anomalies {}", duplicateIds);
 
-    if (!duplicateIds.isEmpty()) {
-      removeDuplicates(duplicateIds, startTime);
-    }
+    //    if (!duplicateIds.isEmpty()) {
+    //      removeDuplicates(duplicateIds, startTime);
```
Comment (hitesharinga) [inline 280-batch-processing/src/main/java/io/harness/batch/processing/anomalydetection/RemoveDuplicateAnomaliesTasklet.java:48]: enable it again..

File: 400-rest/src/main/java/io/harness/ccm/anomaly/dao/AnomalyEntityDao.java [hunk 2]
```diff
@@ -276,9 +285,9 @@ public void insert(List<? extends AnomalyEntity> anomaliesList) {
             statement.addBatch(getInsertQuery(anomaly));
             index++;
             if (index % BATCH_SIZE == 0 || index == anomaliesList.size()) {
-              log.debug("Prepared Statement in AnomalyEntityDao: {} ", statement);
+              log.info("Prepared Statement in AnomalyEntityDao: {} ", statement);
```
Comment (hitesharinga) [inline 400-rest/src/main/java/io/harness/ccm/anomaly/dao/AnomalyEntityDao.java:288]: change this.

**Comment by hitesharinga** [comment]
trigger messagecheck

**Review by nikunj-badjatya — APPROVED** [review]
> Reviewed 23 of 25 files at r1, 3 of 3 files at r2, 3 of 7 files at r4, 6 of 6 files at r5, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/33224)* status: :shipit: complete! ✓

**Pull request merged**